submissions/no pain no gain/no pain no gain.md

```diff
@@ -0,0 +1,78 @@
+
+![](https://www.youtube.com/watch?v=2MnXgqCxMIg)
+
+To make this kind of "cinematic" videos is good to know [factorio console](https://wiki.factorio.com/Console) and [map editor](https://wiki.factorio.com/Map_editor). Because it gives you ability to build big factory in no time, modify researches and for example speed up time. There are many great commands what you can use and whole list is on [factorio wiki](https://wiki.factorio.com/Console). It is also interesting, that Factorio trailers used special prepared [Lua scripts](https://www.lua.org/about.html), like we can read in description: "All the actions, movement and logic in general in the trailer are controlled by the game native modding system (Lua scripting)".
+
+![](https://i.imgur.com/BDoJjko.gif)
+
+Whole necessary [factorio API](https://lua-api.factorio.com/latest/) needed to work with Lua is well described on official website.
+
+![](https://i.imgur.com/q9qlUXa.gif)
+
+With the help of the above-mentioned technics, I've build very interesting machines that are useless in every aspect and their only purpose was to deal maximum amount of pain. There is a small list with explanation.
+I wanted to show cumulative "Train out of fuel" alert and to do this, I simply stopped time in editor and make 10 pointless rails circles. In every circle, I've placed one locomotive and setup schedule for infinite repetitions. As time was stopped, there was no problem to put one more wood to each subsequent locomotive. When everything was ready, I exited editor mode to start time and recorded appearing pop-ups.
+
+![](https://i.imgur.com/i40EZ8C.gif)
+
+Second machine was used to make yt thumbnail and it's just four locomotives connected to one circuit condition and set up to travel straight no matter what. You can also see simple self-timer made out of belts to have time for taking best position.
+
+![](https://i.imgur.com/E2Gatvk.gif)
+
+### No pain, No gain
+Factorio has come a long way (8.5 years) to be right where it is. Mistakes were made, but devs didn't pretend that everything is fine and they work hard to make factorio a wonderful games for casual players and veterans. From this submission you can think than I'm tormenting this game and looking for everything I can attach to, but truth is I respect work that Wube software [team](https://factorio.com/game/about) put into this masterpiece.
+
+In my next submission I'll try to explain why factorio is such a perfect game instead of saying what is wrong I'll say what is good. I'm not a member of Wubue software, but my knowledge and experience is enough to revers-engineer the best parts of the game. It will only be the beginning of the iceberg, because 8.5 years of development can't be shrink to one little post.
```

Comment:
Not sure this part is neccesary, when the next part comes out, we can refer back to this one instead i think.